### PR TITLE
docs: adding patches and integrations v2

### PR DESCRIPTION
**Pull request by taku-nm**
changed as per osu request

### CODE REVIEW

File: docs/0_prerequisites.md [hunk 1]
```diff
@@ -7,9 +7,12 @@ To use ReVanced CLI, you will need to fulfill certain requirements.
 - Java SDK 11 (Azul JDK or OpenJDK)
 - [Android Debug Bridge (adb)](https://developer.android.com/studio/command-line/adb) if you want to deploy the patched APK file on your device
 - An ABI other than ARMv7 such as x86 or x86-64 (or a custom AAPT binary that supports ARMv7)
+- Revanced Patches
+- Revanced Integrations if your patches require them
```
Comment (oSumAtrIX) [inline docs/0_prerequisites.md:11]:
```suggestion
- ReVanced Patches
- ReVanced Integrations, if the patches require it
```

File: docs/0_prerequisites.md [hunk 2]
```diff
@@ -7,9 +7,12 @@ To use ReVanced CLI, you will need to fulfill certain requirements.
...
+
 
 ## ⏭️ Whats next
 
-The next section will show, how to use [ReVanced CLI](https://github.com/revanced/revanced-cli).
+The next section will show you how to use ReVanced CLI
 
-Continue: [🛠️ Using ReVanced CLI](1_usage.md)
+Continue to: [🛠️ Using ReVanced CLI](1_usage.md)
```
Comment (oSumAtrIX) [inline docs/0_prerequisites.md:18]:
```suggestion
Continue: [🛠️ Using ReVanced CLI](1_usage.md)
```

File: docs/0_prerequisites.md [hunk 3]
```diff
@@ -7,9 +7,12 @@ To use ReVanced CLI, you will need to fulfill certain requirements.
 - Java SDK 11 (Azul JDK or OpenJDK)
 - [Android Debug Bridge (adb)](https://developer.android.com/studio/command-line/adb) if you want to deploy the patched APK file on your device
 - An ABI other than ARMv7 such as x86 or x86-64 (or a custom AAPT binary that supports ARMv7)
+- ReVanced Patches
+- ReVanced Integrations, if the patches require it
+
```
Comment (oSumAtrIX) [inline docs/0_prerequisites.md:12]:
```suggestion
```

**Comment by Ushie** [comment]
Balls

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**